src/api/wayfire/core.hpp

```diff
@@ -155,10 +155,17 @@ class compositor_core_t : public wf::object_base_t
      */
     virtual wf::surface_interface_t *get_touch_focus() = 0;
 
+    /**
+     * @return The surface at point, or null if none.
```

Comment:
Both functions are missing comments about the coordinate systems of their parameters (output-local, output-layout ..)